docs/controls/src/imageviewer-all/ImageViewerWindow.qml

```diff
@@ -0,0 +1,59 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.4
+import QtQuick.Dialogs 1.2
+
+ApplicationWindow {
+
+    // ...
+
+    function openFileDialog() { fileOpenDialog.open(); }
+    function openAboutDialog() { aboutDialog.open(); }
+
+    visible: true
+    title: qsTr("Image Viewer")
+
+    background: Rectangle {
+        color: "darkGray"
+    }
+
+    Image {
+        id: image
+        anchors.fill: parent
+        fillMode: Image.PreserveAspectFit
+        smooth: true
```

Comment:
true is the default for smooth, so this could be left out.

Reply:
taken!